barretenberg/sol/src/honk/HonkTypes.sol

```diff
@@ -146,4 +147,36 @@ library Honk {
         Honk.G1ProofPoint shplonkQ;
         Honk.G1ProofPoint kzgQuotient;
     }
+
+    struct ZKProof {
+        uint256 circuitSize;
+        uint256 publicInputsSize;
+        uint256 publicInputsOffset;
+        // Commitments to wire polynomials
+        Honk.G1ProofPoint w1;
+        Honk.G1ProofPoint w2;
+        Honk.G1ProofPoint w3;
+        Honk.G1ProofPoint w4;
+        // Commitments to logup witness polynomials
+        Honk.G1ProofPoint lookupReadCounts;
```

Comment:
does the ordering of this need to be different to the Proof above

Reply:
it is roughly in the correct order except the first libra commitment is received before sumcheck, but because the additional subprotocol operates on all at once I thought it's better to group them together